lnbits/core/templates/core/wallet.html

```diff
@@ -569,7 +569,10 @@ <h6 class="text-subtitle1 q-mt-none q-mb-sm">
             ></q-spinner>
           </q-form>
         </q-card>
-        <q-card v-else class="q-pa-lg q-pt-xl lnbits__dialog-card">
+        <q-card
+          v-else-if="receive.paymentReq && receive.lnurl == null"
```

Comment:
the corresponding `v-if` is:
```python
        <q-card
          v-if="!receive.paymentReq"
```
so `receive.paymentReq` is not required in this condition